_tests/browser_tests/environment.js

```diff
@@ -0,0 +1,28 @@
+// Common configuration files with defaults plus overrides from environment vars
+var webServerDefaultPort = 7000;
+
+module.exports = {
+  // The address of a running selenium server.
+  seleniumAddress:
+    (process.env.SELENIUM_URL || 'http://localhost:4444/wd/hub'),
+
+  // Capabilities to be passed to the webdriver instance.
+  capabilities: {
+    'browserName':
+        (process.env.TEST_BROWSER_NAME || 'chrome'),
```

Comment:
Yeah, I picked that up from a tutorial I was reading yesterday. I mostly used it so we could set the default host and use the built in baseUrl param instead of the one I created, which required a full `--params.baseURL` instead of just `--baseUrl`. `browser` isn't available to the config either, so it meant we had to call that param on every Page Object instead of just once in the config. Very cool overall.